### PR TITLE
Add target image storage account sku argument

### PR DESCRIPTION
**Pull request by erezs80**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
image copy

**Comment by qwordy** [comment]
Thank you for your contribution!

**Comment by qwordy** [comment]
Can you update src/image-copy/azext_imagecopy/tests/latest/test_image_copy_commands.py to test the new feature? Thanks.

**Comment by erezs80** [comment]
> Can you update src/image-copy/azext_imagecopy/tests/latest/test_image_copy_commands.py to test the new feature? Thanks.

Done

**Comment by erezs80** [comment]
@qwordy , Is there anything else missing?

File: src/image-copy/azext_imagecopy/tests/latest/test_image_copy_commands.py [hunk 1]
```diff
@@ -27,5 +27,5 @@ def test_image_copy(self, resource_group):
         self.cmd('image create -g {rg} -n {image} --source {vm}')
         self.cmd('group create -g {rg2} -l eastus')
         self.cmd('image copy --source-object-name {image} --source-resource-group {rg} '
-                 '--target-location eastus --target-resource-group {rg2} --target-name {image}')
+                 '--target-location eastus --target-resource-group {rg2} --target-name {image} --target_sku Premium_LRS')
```
Comment (qwordy) [inline src/image-copy/azext_imagecopy/tests/latest/test_image_copy_commands.py:30]:
```suggestion
                 '--target-location eastus --target-resource-group {rg2} --target-name {image} --target-sku Premium_LRS')
```

**Comment by qwordy** [comment]
I run the test and get an error "json.decoder.JSONDecodeError: Extra data: line 33 column 1 (char 1495)". I am debugging it.

**Comment by erezs80** [comment]
> I run the test and get an error "json.decoder.JSONDecodeError: Extra data: line 33 column 1 (char 1495)". I am debugging it.

Hi @qwordy , Did you manage to debug it?

**Pull request closed without merging**